learning-bitmark/property-system-introduction/problem-we-are-trying-to-solve.md

```diff
@@ -0,0 +1,47 @@
+
+That needs to change.
+
+## The Power of Property Systems
+
+To be able to meaningfully exert property rights requires a robust property system that can define and delineate both what assets are and what rights are associated with those assets. Such a property system is powerful because of the many benefits it can offer.
+
+Most obviously, a property system creates frameworks. It creates an organizational framwork that allows us to itemize the things that we own and assemble them into categories that our minds can easily comprehend; it also creates a legal framework that allows anyone to participate in the acquisition and management of property, something that would be monopolized by the powerful elite without such a system.
+
+A property system also creates economic opportity. It allows us to gain benefits from what we own. Perhaps we never plan to transfer our property, and in that case we can guarantee our continued ability to enjoy a property's core values by asserting our rights and preventing theft. But we can also use a property system to offer our property for loan, license, or sale in marketplaces. 
+
+Finally, a property system benefits our entire society. It creates social inclusion by making it easy to register property rights and  to find other people with interest in similar properties; it  promotes environmental stewardship by normalizing respect for property; and it supports societal improvement by making it simple for a community to find who owns properties of importance.
+
+By creating a regularized and well-known property system we simultaneously increase the value of a property and decrease the costs of dealing with it.
+
+All of these advantages are magnified as a property system grows larger, making it easier to find more properties, to create larger communities, and to sell to larger marketplaces. Thus, the most successful property system is not a local property system and not a national property system, but a global one.
+
+## Bitmark's Property System
+
+The Bitmark Property System is the answer to these needs: the way to replicate the property rights of the physical world in a global property system for the digital world. It's _the_ property system for establishing value and legal control over the whole world’s data.
+
+* It allows you to **register** your digital assets and data, defining what the assets are in a special asset record, then issuing them to yourself as the initial owner.
+* It also allows you to **transfer** your digital assets and data. You can sell an original asset or license a digital copy. You can make a personal transfer or do so as part of the Bitmark Markets -- which include marketplaces for specific digital properties.
+* Finally, for cases where ownership is shared, it allows you to **divide** a digital asset into shares, then **grant** and **swap** those shares.

```

Comment:
I am confused by this statement because we technically don't divide a `digital asset` but `a bitmark` into shares. With an asset, it could be issued to a lot of bitmarks. I am not sure if I go too deep into technical here though.